### ISSUE TITLE
Steps to reproduce the reported RPA accuracy of 98%

### ISSUE THREAD

**Issue by anthonio9** [interactiveaudiolab/penn#15]
Hi, 

I'm working on a project similar to yours, but solely focused on guitar pitch recognition. To have a better look into `penn` models training I've integrated [Weights & Biases](https://wandb.ai/site) into the project, checkout [my fork](https://github.com/anthonio9/penn). I'm pretty sure the only thing I've changed in the config is the `LOG_INTERVAL`  value, by setting it to **500**, however the training and validation accuracy reported during the training oscillate around 50%, similar results are reported by the evaluation done after the model is trained. 

The figures below are the result of my take at training the `fcnf0++` model from scratch. 

Training accuracy reported every epoch
![image](https://github.com/interactiveaudiolab/penn/assets/66279891/82bcc9fc-e86f-4911-a05c-8561b2cd7c7e)

Validation accuracy reported every epoch
![image](https://github.com/interactiveaudiolab/penn/assets/66279891/482efd51-7813-4c00-acad-163c22135307)

Training loss
![image](https://github.com/interactiveaudiolab/penn/assets/66279891/28f26b7b-4510-49c9-9918-f15ac80fa203)

The estimated performance is included in the `overall.json` file generated by the training script.

It's clear that I'm missing something, do you have any advice on steps to achieve the best results, perhaps some issues in my take that are obvious? I tried to follow the README instructions, download, preprocess and partition the **mdb** and **ptdb** datasets according to `fcnf0++` config, then run the training.  In the `overal.json` file it's reported that evaluation on **mdb** reaches around 60% and **ptdb** only around 20%. 

[overall.json](https://github.com/interactiveaudiolab/penn/files/13812666/overall.json)

**Comment by anthonio9** [comment]
My bad, the RCA, RPA and RMSE metrics are as reported in the paper, it was just my mistake in reading the evaluation results. Thanks for your great papers and code!
